### PR TITLE
Add some PCRE tests

### PR DESCRIPTION
**Pull request by edk0**

### CODE REVIEW

File: tests/minimise/in17.re [hunk 1]
```diff
@@ -0,0 +1 @@
+[aBx]*a
```
Comment (katef) [inline tests/minimise/in17.re]: We already have this as `in0.re`

Comment (edk0) [reply tests/minimise/in17.re]: I'm not sure how this ended up here; removed.

Comment (katef) [reply tests/minimise/in17.re]: It's getting to be a famous test case now :)

File: tests/pcre/in16.re [hunk 2]
```diff
@@ -0,0 +1 @@
+[[:alpha:]]
```
Comment (katef) [inline tests/pcre/in16.re]:
```
build/bin/re -r pcre -py tests/pcre/in16.re  > build/tests/pcre/got16.fsm
tests/pcre/in16.re:1: Syntax error: expected group body
```
what's going on here?

Comment (edk0) [reply tests/pcre/in16.re]: That's the effect of pre-#75. I'll rebase so the PR head builds.

Comment (katef) [reply tests/pcre/in16.re]: Oh of course. silly me.

**Pull request merged**